packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -797,10 +803,20 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
                     this.pan(PAN_RIGHT);
                     break;
                 }
+                case 'Shift': {
+                    this.setState({ cursor: 'crosshair' });
+                    break;
+                }
             }
         }
     }
 
+    private onKeyUp(key: React.KeyboardEvent) {
+        if (key.key === 'Shift') {
+            this.setState({ cursor: 'default' });
```

Comment:
It should not reset the cursor is selection is in progress with the left mouse button being pressed.

Reply:
Fixed.